docs/concepts/work-pools.md

```diff
@@ -570,3 +570,5 @@ The Prefect CLI can install the required package for Prefect-maintained worker t
 ### Additional resources
 
 See how to daemonize a Prefect worker in [this guide](/guides/deployment/daemonize/).
+
+For more information on overriding a work pool's job variables see this [guide](/guides/deployment/overriding-job-variables/).
```

Comment:
It could be especially resonant to add this to the "Advanced Customization of the Base Job Template" tip.

Reply:
Could do both.

Reply:
either way!

Reply:
Added above.